main.cpp

```diff
@@ -32,28 +32,286 @@ Purpose:  This project continues developing Project3.
  
  */
 
+#include <iostream>
 /*
  copied UDT 1:
  */
+struct Shoe
+{
+    std::string brand { "Nike" };
+    double size { 0 };
+    std::string color { "white" };
+    std::string condition { "good" };
+    std::string securingMethod { "velcro" };
+    double percentOfSoleLeft = 100;
+    bool beingWorn = false;
+    
+    Shoe();
+    ~Shoe()
+    {
+        std::cout << "Shoe Destroyed\n";
+    }
```

Comment:
0) move all of your implementations of all functions to OUTSIDE of the class. 
- [ ] revise